docs/conf.py

```diff
@@ -93,58 +95,88 @@
 # the benefit of other packages who want to refer to objects in the
 # astropy core.  However, we don't want to cyclically reference astropy in its
 # own build so we remove it here.
-del intersphinx_mapping['astropy']  # noqa: F405
+del intersphinx_mapping["astropy"]  # noqa: F405
 
 # add any custom intersphinx for astropy
-intersphinx_mapping['astropy-dev'] = ('https://docs.astropy.org/en/latest/', None)  # noqa: F405
-intersphinx_mapping['pyerfa'] = ('https://pyerfa.readthedocs.io/en/stable/', None)  # noqa: F405
-intersphinx_mapping['pytest'] = ('https://docs.pytest.org/en/stable/', None)  # noqa: F405
-intersphinx_mapping['ipython'] = ('https://ipython.readthedocs.io/en/stable/', None)  # noqa: F405
-intersphinx_mapping['pandas'] = ('https://pandas.pydata.org/pandas-docs/stable/', None)  # noqa: E501, F405
-intersphinx_mapping['sphinx_automodapi'] = ('https://sphinx-automodapi.readthedocs.io/en/stable/', None)  # noqa: E501, F405
-intersphinx_mapping['packagetemplate'] = ('https://docs.astropy.org/projects/package-template/en/latest/', None)  # noqa: E501, F405
-intersphinx_mapping['h5py'] = ('https://docs.h5py.org/en/stable/', None)  # noqa: F405
-intersphinx_mapping['asdf-astropy'] = ('https://asdf-astropy.readthedocs.io/en/latest/', None)  # noqa: F405
-intersphinx_mapping['fsspec'] = ('https://filesystem-spec.readthedocs.io/en/latest/', None)  # noqa: F405
+intersphinx_mapping["astropy-dev"] = (  # noqa: F405
```

Comment:
I don't like the `intersphinx_mapping` changes. You cannot grep it and look at a certain package mapping anymore without opening the file. But maybe it is just me?

Reply:
I'll revert this to the original and add `# fmt: off`/`# fmt: on`.